text/src/autogluon/text/automm/utils.py

```diff
@@ -594,6 +612,19 @@ def create_model(
                 normalization=model_config.normalization,
                 num_classes=num_classes,
             )
+        elif model_name.lower().startswith(CATEGORICAL_TRANSFORMER):
```

Comment:
We need to add the new models' configs into `automm/configs/model/fusion_mlp_image_text_tabular.yaml`. Since we are adding more models, the original yaml file name `fusion_mlp_image_text_tabular.yaml` can be changed to `default.yaml`.